Marlin/Configuration.h

```diff
@@ -884,6 +884,7 @@ const bool Z_MIN_PROBE_ENDSTOP_INVERTING = false; // set to true to invert the l
 // leaving it undefined or defining as 0 will disable the servo subsystem
 // If unsure, leave commented / disabled
 //
+//#define BLTOUCH_MENU // uncomment if you want the reset BLTouch menu item
```

Comment:
Any changes made to this `Configuration.h` file must be applied to all the other `Configuration.h` files included with Marlin.